### PR TITLE
Include Community annotation

### PR DESCRIPTION
**Pull request by Swaathik**
## Purpose
https://www.ebi.ac.uk/panda/jira/browse/TRM-30488

## Approach
A new component is created to show the annotations and it is displayed at the respective sections

## Testing
Added a test

## Checklist
- [x] My PR is scoped properly, and “does one thing only”
- [x] I have reviewed my own code
- [x] I have checked that linting checks pass and type safety is respected
- [x] I have checked that tests pass and coverage has at least improved, and if not explained the reasons why

### CODE REVIEW

File: src/uniprotkb/components/entry/CommunityCuration.tsx [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+            variant="tertiary"
+            className={styles['community-curation-button']}
+            onClick={() => setToggleView(!toggleView)}
+          >
+            <CommunityAnnotationIcon />
+            {`Community curation (${communityReferences.length}) `}
+            {toggleView ? (
+              <ChevronUpIcon width="1ch" />
+            ) : (
+              <ChevronDownIcon width="1ch" />
```
Comment (aurel-l) [inline src/uniprotkb/components/entry/CommunityCuration.tsx:52]: The only place where you'd need a state would be here, for styling, so as a workaround as the open/closed state is on the html details element then you could just render one chevron, and transform it to flip it in CSS with the selector `[open]` for the parent details element

File: src/uniprotkb/components/entry/CommunityCuration.tsx [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+                      {communityAnnotation?.disease}
+                    </p>
+                  )}
+                  {communityAnnotation?.comment && (
+                    <p>{communityAnnotation?.comment}</p>
+                  )}
+                  <div className={styles['contributor-details']}>
+                    {citationId && (
+                      <span>
+                        Source:&nbsp;&nbsp;
```
Comment (aurel-l) [inline src/uniprotkb/components/entry/CommunityCuration.tsx:84]: Any reason for the double nbsp here and the same a bit below? Is that just for a bit more spacing?

**Review by aurel-l — APPROVED** [review]

File: src/uniprotkb/components/entry/CommunityCuration.tsx [hunk 3]
```diff
@@ -39,81 +39,79 @@ const CommunityCuration = ({
...
-            <hr className={styles.separator} />
-          </summary>
-          <HeroContainer className={styles.content}>
-            {communityReferences.map(
-              ({ source, communityAnnotation, citationId }) => (
-                <div key={source?.id}>
-                  {communityAnnotation?.proteinOrGene && (
-                    <b>{communityAnnotation.proteinOrGene}</b>
+      <details className={styles['community-annotation-details']}>
+        <summary className={styles.header}>
```
Comment (aurel-l) [inline src/uniprotkb/components/entry/CommunityCuration.tsx:43]: Is that class still needed?

Comment (Swaathik) [reply src/uniprotkb/components/entry/CommunityCuration.tsx:43]: you mean the 'header' class?

**Review by aurel-l — APPROVED** [review]

**Pull request merged**